code/modules/mob/living/carbon/life.dm

```diff
@@ -567,6 +568,39 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 		if(drunkenness >= 101)
 			adjustToxLoss(2) //Let's be honest you shouldn't be alive by now
 
+	if(amphetamization) //Amphetamines have effect now!
+		if(amphetamization >= 1)
+			if(prob(5))
+				Dizzy(1)
+
+		if(mind && (mind.assigned_role == "Scientist" || mind.assigned_role == "Research Director"))
+			if(SSresearch.science_tech)
+				if(amphetamization > 2.5)
+					var/adderalpwr = 4 - amphetamization //dose your amphetamine/adderal/meth well!
+					SSresearch.science_tech.add_point_list(list(TECHWEB_POINT_TYPE_GENERIC = ADDERAL_POINTS * adderalpwr)) 
+
+		if(amphetamization > 10)
+			blur_eyes(1)
+
+		if(amphetamization > 20)
+			blur_eyes(5)
+			if(prob(3))
+				confused += 30
+		
+		if(amphetamization > 40)
+			Dizzy(25)
+
+		if(amphetamization > 50)
+			if(prob(33))
+				adjustOrganLoss(ORGAN_SLOT_HEART, 0.1)
+				vomit()
+		
+		if(amphetamization > 60) //You OD'd HARD
+			if(prob(25))
+				vomit() //consider this a blessing
+			else
+				adjustToxLoss(2)
+				adjustOrganLoss(ORGAN_SLOT_BRAIN, 0.2)

```

Comment:
we DEFINITELY don't need to put this on life

Reply:
actaully i think ill be able to scrap this whole code